frontend/src/metabase/core/components/BookmarkToggle/BookmarkToggle.tsx

```diff
@@ -37,6 +37,10 @@ const BookmarkToggle = forwardRef(function BookmarkToggle(
     setIsAnimating(false);
   }, []);
 
+  const getIcon = () => {
+    return isBookmarked ? "bookmark_filled" : "bookmark";
+  };
```

Comment:
I'd say it's more practical just to make this a variable i.e.

```suggestion
  const icon = isBookmarked ? "bookmark_filled" : "bookmark";
```

This is because functions are supposed to be called by others with some of these reasons
1. The function is called at a different time than its instantiation.
2. You expect the caller to pass some arguments to the function.
3. You don't want the value to be evaluated when the function is being declared e.g. you could reference a variable that's being instantiated after the declaration of a function. Because the function will be called after the variable is instantiated, it would have value. This wouldn't work if you evaluated the variable right away.

There could be more reasons, but these are some reasons that are perfectly valid for putting things in a function. Other than that, if both functions and variables work, I'd prefer the variables first. Like in this case, you could think of it like this. Why do I need a function when I'll immediately call it immediately below its declaration?